guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,384 @@
+# Updating from v12 to v13
+
+## Before you start
+
+v13 requires Node 14.x or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
+
+Once you've got Node up-to-date you can install v13 by running `npm install discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/opus` for voice support.
+
+You can check your discord.js version with `npm list discord.js`. Should it still show v12.x uninstall (`npm uninstall discord.js`) and re-install discord.js and make sure the entry in your package.json does not prevent a major version update. Please refer to the [npm documentation](https://docs.npmjs.com/files/package.json#dependencies) for this.
+
+## Commonly used methods that changed
+
+### Intents
+
+As v13 makes the switch to Discord API v8, it will now be **required** to specify intents in your Client constructor.
+For more information, refer to our more [detailed article about this topic](/popular-topics/intents).
+
+:::danger
+
+`clientOptions.disableMentions` has been removed and replaced with `clientOptions.allowedMentions`!
+
+The Discord API now allows bots much more granular control over mention parsing, down to the specific ID if desired. Refer to the [Discord API documentation](https://discord.com/developers/docs/resources/channel#allowed-mentions-object) for more information.
+
+```diff
+- const client = new Discord.Client({ disableMentions: 'everyone' });
++ const client = new Discord.Client({ allowedMentions: { parse: ['users', 'roles'], repliedUser: true } });
+```
+
+:::
```

Comment:
why is this in the Intents section?

Reply:
Its just missing a heading. The intents "section" is that one paragraph.